udata_front/frontend/helpers.py

```diff
@@ -211,6 +218,18 @@ def owner_name_acronym(obj):
     return ''
 
 
+@front.app_template_global()
+def organization_type(org: Organization):
+    if org.public_service:
```

Comment:
Should local authority be prioritized over public service when showing logo and having both badges? cc @agarrone

Reply:
My understanding was that a local authority shouldn't have public service in their badge array. 
Also, we can change the condition but this one is only a "translation" of the existing one in datagouv/components so a new release will be necessary to have the change everywhere.

Reply:
TS change done in https://github.com/datagouv/udata-front/pull/475 too

Reply:
I know we'll treat the root cause in https://github.com/datagouv/data.gouv.fr/issues/1433 but I think the order should match the one in datagouv/components `useOrganizationType`? Else, we may have discrepancy before applying a migration as in http://dev.local:7000/fr/organizations/grand-chambery/#/datasets.